webpack/components/WithOrganization/withOrganization.js

```diff
@@ -1,9 +1,20 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
 import { translate as __ } from 'foremanReact/common/I18n';
 import { get } from 'lodash';
 import SetOrganization from '../SelectOrg/SetOrganization';
 import Header from '../../containers/Application/Headers';
+import * as organizationActions from '../../scenes/Organizations/OrganizationActions';
+
+const mapStateToProps = state => ({
+  organization: state.katello.organization,
+});
+
+const actions = { ...organizationActions };
+const mapDispatchToProps = dispatch => bindActionCreators(actions, dispatch);
```

Comment:
Is there any real need for this instead of just using `bindActionCreators({ ...organizationActions }, dispatch);`?

Reply:
either way works for me, I can update to one line :+1: